test/11-api-binder.spec.ts

```diff
@@ -307,6 +307,51 @@ describe('ApiBinder', () => {
 		});
 	});
 
+	describe('local mode', () => {
+		const components: Dictionary<any> = {};
+
+		before(() => {
+			return initModels(components, '/config-apibinder.json');
+		});
+
+		after(async () => {
+			// @ts-ignore
```

Comment:
Can you use `ts-expect-error` with information about why/what you're expecting please?

Reply:
I see this was just from an extra file moved here. Feel free to ignore then.